Chore update docs (#92)
vtex-apps/store-sitemap: docs/README.md

:warning: *Before generating your store's sitemap, you might want to adjust if products, navigation, app and/or custom routes will be included in it or not. If that's the case, check the Advanced Configuration section for more information before proceeding any further.*

4. Open GraphQL IDE in the admin and select `vtex.routes-bootstrap@0.x`

:information_source: If there isn't an `GraphQL IDE` option under `Store Setup`, you need to install the app: `vtex.admin-graphql-ide@3.x`

5. Run the folowwing query: 
```
{
  bootstrap {
    brands
    categories
  }
}
```

> In the response, you'll get the containing information about the number of routes that were saved in the database. The department, categories and subcategories routes will be under the `category` property and number of brand routes under the `brands` property.
> 
> If this query has already been run and there isn't any change it these types of routes you don't need to run again.

Reviewer: ```suggestion

The expected response is a `JSON` object containing the number of brand routes under the `brands` property; and the number of department, category, and subcategory routes under `categories`.
```

> 6. Open GraphQL IDE in the admin and select `vtex.store-sitemap@2.x`
> 
> 7. Run the folowwing query: 

Reviewer: ```suggestion
8. Now, from the GraphQL IDE dropdown list, choose the `vtex.store-sitemap@2.x` app.

9. Run the following query: 
```

```
{
  generateSitemap
}
```

> The expected response body is an `OK` in text format. This means your sitemap will be available in some minutes, after being processed and saved on our database.

Reviewer: A resposta vai ser um JSON ou vai continuar sendo isso mesmo?

Author: Não, vai ser um booleano agora

Reviewer: Assim?
```
{
  "data": {
    "generateSitemap": true
  }
}
```

Author: Exato!

Next generation available: <End-date>	
```	

> *To make a force restart, add the `force` argument to the query like: `generateSitemap(force: true)`. But, be aware that this will cancel the previous process.*	

Reviewer: ```suggestion
*To make a force restart, add the `force` argument to the query, as in: `generateSitemap(force: true)`. But, be aware that this will cancel the previous process.*	
```

> 9. Check the sitemap generated for the current workspace you are working on by accessing `https://{workspace}--{account}.myvtex.com/sitemap.xml` on your browser. Notice that if your store is a cross-border one, you'll first see an index containing a website's sitemap for each locale.

Reviewer: ```suggestion
10. Check the sitemap generated for the current workspace you are working on by accessing `https://{workspace}--{account}.myvtex.com/sitemap.xml` on your browser. Notice that if your store is a cross-border one, you'll first see an index containing a website's sitemap for each locale.
```
